esphome/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_ESPHOME_IP
+      APP_PORT: 6052
+      PROXY_AUTH_ADD: "true"
+
+  server:
+    image: esphome/esphome:2022.6.2@sha256:69d382af56b10006a833eb90d29339b12efdddfb93382764a379c17fe252eda6
```

Comment:
This appears to have the digest for architecture ARM64. The digest should be: `sha256:1fac6334b85e76c7fd936036891b7a7ee761ee73fd119662b2cb564c3faf818f`

You can get a multi-arch. digest by running: `docker pull <image>` and then you'll see the digest at the bottom.